browser-based-ingestion/dist/index.html

```diff
@@ -0,0 +1,110 @@
+     "Auth": 
+      {"Type": "BASIC",
+       "Username": "foo",
+       "Password": "foo"},
+     "VpcConfiguration": 
+      {"SecurityGroupIds": 
+        ["foo", "foo"],
+       "SubnetIds": 
+        ["foo", "foo"],
+       "DeviceCidrBlocks": 
+        ["foo", "foo"]
+      }
+    },
+   "RoleARN": "foo"}
+}
+                    </textarea><br>
+
+                    <input type="radio" id="uploadFile" name="dataFeedSelection" value="uploadFile" class="user-information"><label>Upload a file</label>
+                    <input type="radio" id="recordWebcam" name="dataFeedSelection" value="recordWebcam" class="user-information"><label>Record using Webcam</label><br>
+
+                    <div class="hide user-information" id="fileUploadRequirements" style="border:1.3px solid black; padding: 20px">                    
+                        <input type="file" name="inputFile" id = "inputFile"><br>
+                        <input type="radio" id="mkv" name="containerSelection" value="mkv-h264" class="user-information" checked><label>MKV/WebM with H264 video codec</label><br>
+                        <input type="radio" id="other" name="containerSelection" value="other" class="user-information"><label>Other</label><br>
+                        <label>Note: Other video types require transcoding of the video frames to H264 and it can be time-consuming</label>
+                    </div>
+
+                    <div class="hide user-information" id="videoRecordRequirements" style="border:1.3px solid black; padding: 20px">
+                        <input type="checkbox" id="video"  class="user-information" checked disabled><label>Video</label><br>
+                        <input type="checkbox" id="audio"  class="user-information" checked><label>Audio</label><br>
+                        <label>Height (in pixels):</label><input type="number" id="height" placeholder="Enter height" class="user-information"><br>
+                        <label>Width (in pixels):</label><input type="number" id="width" placeholder="Enter width" class="user-information"><br>
+                        <label>Frequency of putMediaAPI calls (in milliseconds):</label><input type="number" id="latency" placeholder="Enter latency" class="user-information"><br>
+                        <label>Frame rate:</label><input type="number" id="frameRate" placeholder="Enter frame rate" class="user-information"><br>
+                    </div>
+
+                    <input type="button" id="startRecord" value="Send Data to PutMedia" class="user-information">
+                    <input type="button" id="stopRecord" value="Stop Recording" class="hide user-information" disabled><br>
+                </div>
+                
+            </div>
+        </div>
+        <div class="wrapper">
+            <video id="webcam" width="640px" height="480px"></video>
+        </div>
+    </body>
+
+    <script src="https://unpkg.com/@ffmpeg/ffmpeg@0.8.3/dist/ffmpeg.min.js"></script>
```

Comment:
Can we get these via npm?

Reply:
I could not figure out how to do that with `npm` but now I have downloaded the file to a `lib` folder in my project, so there is no third-party hosting anymore.